person.rb

```diff
@@ -7,15 +7,23 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @corrector = Corrector.new
   end
 
   def of_age?
     @age >= 18
   end
 
+  def validate_name
+    @name = @corrector.correct_name
```

Comment:
- [ ] Currently, validate_name is not working as expected. I think you should pass `@name` as an argument because the function `correct_name` is expecting one argument. Kindly fix the issue.

Reply:
Thanks for the feedback, issue resolved